### DISCUSSION TITLE
With Search-Order word set should `FIND-NAME` be extended similar to `FIND`?

### DISCUSSION THREAD

**Discussion by SirWumpus** [ForthHub/discussion#172] [Q&A] [answered]
In Forth 200x draft 19-1 section 16.6.1 there is no mention of `FIND-NAME` being extended in a similar manner to that of `FIND`.  I assume this is just some over sight due to the recent addition of `FIND-NAME`.

**Comment by ruv** [comment, marked as answer]
There is no need to update the text description for `FIND-NAME` in the section 16 "The optional Search-Order word set" because that text description is already given in terms of "search order".

I think, the "search order" term should be moved to the section 2, and the text description for `FIND` should also be given using that term, and then an update for `FIND` in the section 16 would not be necessary.
